### DISCUSSION TITLE
Command line arguments: Use both script and expression

### DISCUSSION THREAD

**Discussion by ginabythebay** [sharkdp/numbat#353] [Ideas]
It would be handy to be able to reference a script from the command line that would, for instance, define a function, and then call that function, also from the same command line.

See also #258

**Comment by sharkdp** [comment]
I converted it to a ticket: #354. Please note that you can already put common definitions in a `~/.config/numbat/init.nbt` file and use them with `-e`/`--expression`. Does that help for your use case?

**Reply by ginabythebay** [reply]
That definitely helps my use case, thank you! Sorry I remember you talking about a ticket in the other GitHub discussion and didn’t understand you were using it as a synonym for issue.

**Reply by sharkdp** [reply]
#354 is now implemented as well.
